Add intro to hardware ready section (#37)
eficode-academy/On-prem_Kubernetes_Guide: docs/hardware_ready/_index.md

---
## Decision Matrix

> This is a badass intro, that I have put here to demo being brilliant at Git for some students.

Reviewer: ```suggestion
This is an awesome intro, that I have put here to demo being brilliant at Git for some students!!

```
